lib/boot.js

```diff
@@ -33,8 +33,8 @@ const cookie = parse(document.cookie);
 const auth = new Auth(config.app_id, config.app_key);
 const appProvider = 'simplenote.com';
 
-const token = cookie.token || localStorage.access_token;
 const appID = config.app_id;
+let token = cookie.token || localStorage.access_token;
```

Comment:
By the way do you know if this cookie is only here for legacy reasons? (Can we drop it...?)

Reply:
The cookie is set by app engine when signing in on the web. We could probably have it pass it directly instead of using a cookie? We can probably sort that out in a separate PR.

Reply:
Oh ok, thanks for clarifying 👍